axelrod/tests/unit/test_tournament.py

```diff
@@ -49,7 +49,8 @@ def test_init(self):
             processes=4,
             noise=0.2)
         self.assertEqual(len(tournament.players), len(self.players))
-        self.assertEqual(tournament.players[0].tournament_length, 200)
+        self.assertEqual(
+            tournament.players[0].tournament_attributes['length'], 200)
```

Comment:
Could we include a test here for the game attribute also?

And perhaps include tests for non default values?